src/game_renderer/ViewFrustumCulling.h

```diff
@@ -0,0 +1,20 @@
+// Alex Ottoboni
+#ifndef __VIEW_FRUST_CULLING__
+#define __VIEW_FRUST_CULLING__
+
+#define LEFT 0
+#define RIGHT 1
+#define BOTTOM 2
+#define TOP 3
+#define NEAR 4
+#define FAR 5
```

Comment:
I would consider adding a prefix or something to these macros since they will be available in all files that include this header, but it doesn't really matter